introduction.md

```diff
@@ -1 +1,7 @@
 hi, I am Zach, nice to meet everyone in the Dojo! My favorite path is the Javascript path
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?